packages/dev/query/src/index.js

```diff
@@ -24,6 +24,7 @@ export function loadGraphs(cacheDir: string): {|
   bundleGraph: ?BundleGraph,
   requestTracker: ?RequestTracker,
   bundleInfo: ?Map<ContentKey, PackagedBundleInfo>,
+  cacheInfo: ?Map<string, Array<string | number>>,
```

Comment:
This parameter doesn't seem to be used

Reply:
Not sure I understand what's wrong, sorry.`cacheInfo` is meant to store the the size and deserialize times of the loaded graphs, and then returned from the `loadGraphs` function to be used in the `cli` function. It should be partially populated during `loadGraph`.

Reply:
Oops, I missed some punctuation and saw this 🙈 
```js
export function loadGraphs(cacheDir: string,
  assetGraph: ?AssetGraph,
  bundleGraph: ?BundleGraph,
  requestTracker: ?RequestTracker,
  bundleInfo: ?Map<ContentKey, PackagedBundleInfo>,
  cacheInfo: ?Map<string, Array<string | number>>,
) {
	...
	let cacheInfo: ?Map<string, Array<string | number>> = new Map();
```